crates/modelardb_server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -1065,6 +1068,78 @@ mod tests {
             vec![TypeId::of::<ParquetExec>(), TypeId::of::<ParquetExec>()],
         ];
 
-        common_test::assert_eq_physical_plan_expected(physical_plan, &expected_plan);
+        assert_eq_physical_plan_expected(physical_plan, &expected_plan);
+    }
+
+    /// Parse, plan, and optimize the `query` for execution on data in `path`.
+    pub async fn query_optimized_physical_query_plan(
```

Comment:
Unclear why `query_optimized_physical_query_plan()` is still public after being moved to `model_simple_aggregates.rs`, maybe either make it private or add a comment or doc comment.

Reply:
Good catch, it has been made private.